src/haven/instrument/robot.py

```diff
@@ -0,0 +1,124 @@
+    ry = Cpt(EpicsMotor, ":ry", labels={ "joint"})
+    rz = Cpt(EpicsMotor, ":rz", labels={ "joint"})
+    acc = Cpt(EpicsMotor, ":acceleration", labels={ "joint"})
+    vel = Cpt(EpicsMotor, ":velocity", labels={ "joint"})
+    
+    # dashboard
+    remote_control = Cpt(EpicsSignal, ":dashboard:remote_control", labels={"dashboard"}, kind="config")
+    program_rbv = Cpt(EpicsSignal, ":dashboard:program_rbv", labels={"dashboard"}, kind="config")
+    installation = Cpt(EpicsSignal, ":dashboard:installation", labels={"dashboard"}, kind="config")
+    play = Cpt(EpicsSignal, ":dashboard:play", labels={"dashboard"}, kind="config")
+    stop = Cpt(EpicsSignal, ":dashboard:stop", labels={"dashboard"}, kind="config")
+    pause = Cpt(EpicsSignal, ":dashboard:pause", labels={"dashboard"}, kind="config")
+    quit = Cpt(EpicsSignal, ":dashboard:quit", labels={"dashboard"}, kind="config")
+    shutdown = Cpt(EpicsSignal, ":dashboard:shutdown", labels={"dashboard"}, kind="config")
+    release_brake = Cpt(EpicsSignal, ":dashboard:release_brake", labels={"dashboard"}, kind="config")
+    close_safety_popup = Cpt(EpicsSignal, ":dashboard:close_safety_popup", labels={"dashboard"}, kind="config")
+    unlock_protective_stop = Cpt(EpicsSignal, ":dashboard:unlock_protective_stop", labels={"dashboard"}, kind="config")
+    restart_safety = Cpt(EpicsSignal, ":dashboard:restart_safety", labels={"dashboard"}, kind="config")
+    program_running = Cpt(EpicsSignal, ":dashboard:program_running", labels={"dashboard"}, kind="config")
+    safety_status = Cpt(EpicsSignal, ":dashboard:safety_status", labels={"dashboard"}, kind="config")
+    power = Cpt(EpicsSignal, ":dashboard:power", labels={"dashboard"}, kind="config")
+    power_rbv = Cpt(EpicsSignal, ":dashboard:power_rbv", labels={"dashboard"}, kind="config")
+    
+    # gripper
+    act = Cpt(EpicsSignal, ":gripper:ACT", labels={"gripper"}, kind="config")
+    acr = Cpt(EpicsSignal, ":gripper:ACR", labels={"gripper"}, kind="config")
+    cls = Cpt(EpicsSignal, ":gripper:CLS", labels={"gripper"}, kind="config")
+    opn = Cpt(EpicsSignal, ":gripper:OPN", labels={"gripper"}, kind="config")
+    rbv = Cpt(EpicsSignal, ":gripper:RBV", labels={"gripper"}, kind="config")
+    val = Cpt(EpicsSignal, ":gripper:VAL", labels={"gripper"}, kind="config")
+    
+    # busy 
+    busy = Cpt(EpicsSignal, ":busy", labels={"busy"}, kind="config")
+    
+    # sample transfer
+    current_sample = Cpt(EpicsSignal, ":current_sample", labels={"transfer"}, kind="config")
+    unload_current_sample = Cpt(EpicsSignal, ":unload_current_sample", labels={"transfer"}, kind="config")
+    current_sample_reset = Cpt(EpicsSignal, ":current_sample_reset", labels={"transfer"}, kind="config")
+    home = Cpt(EpicsSignal, ":home", labels={"transfer"}, kind="config")
+    cal_stage = Cpt(EpicsSignal, ":cal_stage", labels={"transfer"}, kind="config")
+    
+    sample = DCpt(EpicsSignal,transfer_sample(24), labels={"transfer"}, kind="normal")
```

Comment:
Consider renaming from *sample* to *samples* in line with the convention used elsewhere in Haven (e.g. XRF detector ROIs).